Gemfile.lock

```diff
@@ -0,0 +1,56 @@
+GEM
```

Comment:
Is there a reason you committed the `Gemfile.lock`?
Usually you don't have to, as it's generated when `bundle install` is run.

Reply:
I committed it because the bundler folks [suggest](https://bundler.io/v2.0/guides/faq.html#committing-lockfiles-in-libraries) it.

Reply:
That link looks like it is only for gems, actually. And I'm not one.

Reply:
Yeah, I think they recommend it if you develop a gem...though I don't do it = )